FranciscoPerez/react-challenge-project-jan-2023
# Fp 1 fix form submit

## Changes
1. Fix "onChange" event handler function to reference correct input value.

## Purpose
Orders were not being added to the database.

[Issue 1 - Order Form Submit Bug](https://github.com/Shift3/react-challenge-project-jan-2023/issues/1)

## Approach
Change the value referenced in the onChange event handler from `event.value` to `event.target.value`.
This allows the submit action to send the correct input values to the `/api/add-order/` endpoint.

## Testing Steps
1. Open root page (e.g. http://localhost:3000/) and click "Go to Login" button.
3. Click "Login" button.
4. Click "Order Form" nav link.
5. Select menu item.
6. Select quantity.
7. Click "Order it!" button.
8. Click "View Orders" nav link.
9. Verify placed order exists.

## Learning
Troubleshooting was slightly troublesome because an error was not generated for the incorrect value reference.

The issue was evident once I traced through the form submit process.

Closes #1
